### DISCUSSION TITLE
How to render an I-Beam style cursor?

### DISCUSSION THREAD

**Discussion by displexic** [ratatui/ratatui#1054] [Discussion]
Hi. I've been able to position block style cursors just fine, but I wasn't able to find anything on how to render I-Beam style cursors. 

Is there a suggested way on how to go about doing this, if it's possible?

**Comment by orhun** [comment]
Hello! Which backend are you using? e.g. crossterm has [`SetCursorStyle`](https://docs.rs/crossterm/latest/crossterm/cursor/enum.SetCursorStyle.html) method.

**Reply by displexic** [reply]
Thanks! I will have a look into it.

**Reply by hauntedhost** [reply]
@slekup lemme know if you figure this out, i'm interested in doing the same thing
